docs/operator-manual/sso/index.md

```diff
@@ -116,6 +116,10 @@ data:
     cliClientID: vvvvwwwwxxxxyyyyzzzz
 ```
 
+NOTES:
```

Comment:
Can you please use an admonition note block for that, i.e. via `!!! note` (refer to https://squidfunk.github.io/mkdocs-material/extensions/admonition/ or other parts of the docs for details).

Reply:
Done.